liferay-echo/liferay-portal
# LPS-121881 Migrate v2 usages in journal-web

Hi @liferay-echo team. This is a continuation of https://github.com/liferay-echo/liferay-portal/pull/3891, where I missed some management toolbars to migrate. 

The `<clay:management-toolbar-v2 />` tag has been replaced with the
`<clay:management-toolbar />` tag (which uses clay v3).

**Test plan for each of the jsps changed:**

1. Go to Control Menu > Content Data > Web Content
2. Web Content tab: `view.jsp`
3. Structures tab: `view_ddm_structures.jsp`
4. Templates tab: `view_ddm_templates.jsp`
5. Go back to Web Content tab, and select any item and click on "View History" on the kebab menu: `view_article_history.jsp`
6. Create more than 8 structures. On the Web Content tab click on + icon, and then on "Show more" button: `view_more_menu_items.jsp`
7. Enable Feeds (System settings > Web Content > admin > check "Show Feeds". Then go back to Control Menu > Content Data > Web Content and select the "Feeds" tab: `view_feeds.jsp`
8. Edit a page, insert and element and map it to a content: `select_articles.jsp`
![image (2)](https://user-images.githubusercontent.com/8373764/110792525-cf8e6e00-8273-11eb-92e0-8bc3f1131b0a.png)


Let me know if you need any help with this.
Thank you!
